salsah/src/public/index.html

```diff
@@ -179,6 +179,21 @@
 
             SALSAH.ApiPost('session', {username: $('#user_id').val(), password: $('#password').val()}, authorization, function(data) {
 				if (data.status == ApiErrors.OK) {
+					console.log("session created with id " + data.sid);
+					$.ajax({
+						url: "http://localhost:1024/Knora_login",
```

Comment:
Also, can you add some comments to the JavaScript here to explain what this code is doing?